lib/peeps.rb

```diff
@@ -0,0 +1,37 @@
+require 'pg'
+
+  class Peeps
+
+    attr_reader :user_name, :peep
+
+    def initialize(user_name:, peep:)
+     @user_name = user_name
+     @peep = peep
+    end
+
+    def self.connect
+     if ENV['ENVIRONMENT'] == 'test'
+       connection = PG.connect(dbname: 'chitter_manager_test')
+     else
+       connection = PG.connect(dbname: 'peeps_manager')
+     end
+    end
+    def self.all
+      connection = Peeps.connect
+
+      result_peeps = connection.exec( "SELECT * FROM peeps INNER JOIN users ON (peeps.user_id = users.id);" )
```

Comment:
Great that you've extracted the database connection into a new method! You could go one step further and extract it into a database class.